test/development/basic/node-builtins/app/layout.js

```diff
@@ -0,0 +1,12 @@
+export const metadata = {
```

Comment:
not sure why but turbopack cannot render client components if layout.js does not exists, not sure if this should not be checked in then need to fix those separately.

Reply:
It must exist and next.js does automatically create it if it doesn't exist. Seems like the automatic creation isn't implemented for turbopack.